### DISCUSSION TITLE
Weather icon size

### DISCUSSION THREAD

**Discussion by saxonpaul92** [riffnshred/nhl-led-scoreboard#348] [General]
I am using OWM for my weather and in recent days I noticed that the graphic icons that represent the weather (clear skies, rain, etc...) have changed and not for the better.  The icons appear as double pixeled instead of the normal single pixel..  Clear skies are readable but, rain is just junk.  

It is more an annoyance rather than a bug and i was trying to see if the icons had changed but, I think you may be pulling the icon from OWM API.  If you do, do you alter the icon in anyway or just display what is returned.

If you are altering it where do you store the icons?

You can see in the attached jpeg the "broken clouds" icon is double pixeled.

![20211124_222656](https://user-images.githubusercontent.com/20263198/143379766-8c29490f-7d45-4747-acc3-58734212c206.jpg)

BTW, It does not change with the latest upgrades.

**Comment by falkyre** [comment]
Icons are being pulled from a specific font file that really is the only one for conditions that I found that worked and looked ok.  The font code id is coming from OWM (and in Canada from Environment Canada if using Env Canada as the weather provider) and there is a CSV file that contains the font code to font mapping.  The fonts are unicode and the font file places the "icons" in it's unicode range. The font is from here:  https://erikflowers.github.io/weather-icons/

For reference, these are what the icons from OpenWeatherMap would look like:  https://openweathermap.org/weather-conditions#Weather-Condition-Codes-2

Finding fonts that "work" on a 64x32 matrix, that look good and are readable is not the easiest.  This set of fonts gave the best output that I could find.
